kde-apps/okular/okular_kf6-24.12.3.recipe

```diff
@@ -0,0 +1,189 @@
+SUMMARY="Universal document viewer based on KDE Frameworks"
+DESCRIPTION="Okular is a universal document viewer developed by KDE. Okular works on multiple \
+platforms, including but not limited to Linux, Windows, Mac OS X, *BSD, etc.
+
+Features:
+* Supported Formats: PDF, PS, Tiff, CHM, DjVu, Images, DVI, XPS, ODT, Fiction Book, Comic Book, \
+Plucker, EPub, Fax
+* Sidebar with contents, thumbnails, reviews and bookmarks
+* Annotations support"
+HOMEPAGE="https://okular.kde.org"
+COPYRIGHT="2010-2025 KDE Organisation"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://download.kde.org/stable/release-service/$portVersion/src/okular-$portVersion.tar.xz"
+CHECKSUM_SHA256="8d0894dc006b715dec2d8d57c228b8a9d5d9488e3144dd1af93b3b8363e466a0"
+SOURCE_DIR="okular-$portVersion"
+PATCHES="okular-$portVersion.patchset"
+ADDITIONAL_FILES="okular.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="3.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	okular_kf6$secondaryArchSuffix = $portVersion
+	app:Okular$secondaryArchSuffix = $portVersion
```

Comment:
app: doesn't have secondaryArchSuffix

```suggestion
	app:Okular = $portVersion
```

Reply:
Bugger missed that one, I tend to overlook this :)

Reply:
Need to check this once completed on the buildmaster on a clean VM, if this doesn't suffer from KIO issues it can replace the KF5 one. I'll fix this in one go then, thanks for spotting (ps, a few others already slipped in, so on a next version bump those should be corrected, it doesn't throw an error, so for now it's still good.